pkg/controller/agent/pod.go

```diff
@@ -387,22 +381,25 @@ func getAssociatedFleetServer(params Params) (commonv1.Associated, error) {
 	return &fs, nil
 }
 
-func trustCAScript(ver version.Version, caPath string) string {
-	sharedCAPath := "/etc/pki/ca-trust/source/anchors/"
-	updateCmd := "update-ca-trust"
-	if ver.GTE(version.MinFor(7, 17, 0)) {
-		sharedCAPath = "/usr/local/share/ca-certificates"
-		updateCmd = "update-ca-certificates"
-	}
+func trustCAScript(caPath string) string {
+	redhatSharedCAPath := "/etc/pki/ca-trust/source/anchors/"
```

Comment:
I would use the distro name for consistency:
```suggestion
	ubiSharedCAPath := "/etc/pki/ca-trust/source/anchors/"
```

Also those seem to be constants, so maybe:

```
const (
	ubiSharedCAPath    = "/etc/pki/ca-trust/source/anchors/"
	ubiUpdateCmd       = "/usr/bin/update-ca-trust"
	debianSharedCAPath = "/usr/local/share/ca-certificates/"
	debianUpdateCmd    = "/usr/sbin/update-ca-certificates"
)
```